### PR TITLE
Add initial unmocked tests

### PR DESCRIPTION
**Pull request by Bindu-Maharudrappa**

### CODE REVIEW

**Review by graduad — CHANGES_REQUESTED** [review]
Please git ignore following directories:
-  "test_app/integration_tests/flutter-sdk-native-ui/flutter-sdk-native-ui.xcodeproj/project.xcworkspace/xcuserdata"
- "test_app/integration_tests/flutter-sdk-native-ui/flutter-sdk-native-ui.xcodeproj/xcuserdata"

Also please see my inline comment.

File: test_app/integration_tests/end_to_end/api_call_test.dart [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  setUp(() async {
+    tokenString = token;
+  });
+
+  testWidgets('VoxeetSDK: initiliseToken', (tester) async {
+    await dolbyioCommsSdkFlutterPlugin.initializeToken(
+        tokenString, () => Future.value(tokenString));
+  });
+
+  testWidgets('VoxeetSDK: openSession', (tester) async {
```
[the remark this answers is not in the document]
Comment (Bindu-Maharudrappa) [reply test_app/integration_tests/end_to_end/api_call_test.dart:21]: No, it just builds once in the beginning

**Review by graduad — APPROVED** [review]

**Review by graduad — APPROVED** [review]

**Review by graduad — CHANGES_REQUESTED** [review]
We need to figure out why this times out. We cannot merge before that.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -93,13 +95,43 @@ jobs:
...
+        flutter-version: $FLUTTER_VERSION
+    - name: Fetch dependencies
+      run: |
+        flutter pub get
+    - name: Run tests
+      run: |
+        flutter test -r expanded
+    - name: Run pana analyzer
+      run: |
+        ./scripts/score_analyzer.sh
```
Comment (graduad) [inline .github/workflows/ci.yml:125]: We are running both unit tests and pana on the linux build machine. I am not sure we need to run it here as well. And this goes for the Android build on Mac as well. What do you think @Bindu-Maharudrappa and @kbetl-dlb ?

Comment (Bindu-Maharudrappa) [reply .github/workflows/ci.yml:125]: I agree with you. We can remove it from here I suppose.

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -93,13 +92,40 @@ jobs:
...
+    - name: Set up Flutter version 🔧
+      uses: ./.github/actions/flutter-action
+      with:
+        flutter-version: $FLUTTER_VERSION
+    - name: Fetch dependencies
+      run: |
+        flutter pub get
+    - name: Run tests
+      run: |
+        flutter test -r expanded
```
Comment (graduad) [inline .github/workflows/ci.yml:119]: As these are also run in the linux build. Thus we can remove them from here.

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -81,9 +83,6 @@ jobs:
     - name: Run tests
       run: |
         flutter test -r expanded
```
Comment (graduad) [inline .github/workflows/ci.yml:85]: As these are also run in the linux build. Thus we can remove them from here.

**Review by graduad — CHANGES_REQUESTED** [review]

**Review by graduad — APPROVED** [review]

**Pull request merged**